pretyping/reductionops.ml

```diff
@@ -548,6 +548,8 @@ open Primred
 module CNativeEntries =
 struct
 
+  open UnsafeMonomorphic
```

Comment:
Same remark about opening this, but now it's even worse because it's global to a module. I can tolerate the ones in funind because funind is already a mess beyond repair, but we should try to keep Reductionops reasonable.